docs/guides/perl-secure-development-guide.md

```diff
@@ -145,7 +149,7 @@ To help in this shift, CPANSec has put together several [guides](./) intended to
 
 ## License and use of this document
```

Comment:
Should there be something that outlines major changes, perhaps even links to the github history at https://github.com/CPAN-Security/security.metacpan.org/commits/main/docs/guides/cpan-secure-development-guide.md

Actually, I am not against renaming the file, because the history will be lost.